src/main/java/me/coley/j2h/loader/Importer.java

```diff
@@ -0,0 +1,48 @@
+package me.coley.j2h.loader;
+
+import me.coley.j2h.modle.*;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public final class Importer {
+
+    public static Configuration importConfiguration(String path) throws JAXBException, IOException {
+        JAXBContext context = JAXBContext.newInstance(Configuration.class);
+        Unmarshaller um = context.createUnmarshaller();
+        return (Configuration) um.unmarshal(new StringReader(readFile(path, StandardCharsets.UTF_8)));
+    }
+
+    public static Configuration importDefaultConfiguration() throws JAXBException, IOException {
+        return importConfiguration(Importer.class.getClassLoader().getResource("default-config.j2h").getPath());
+    }
+
+    private static String readFile(String path, Charset encoding) throws IOException {
+        byte[] encoded = Files.readAllBytes(Paths.get(path));
+        return new String(encoded, encoding);
+    }
+
+    // For testing Importer only.
+    public static void main(String[] args) throws JAXBException, IOException {
```

Comment:
Demo & Testing code can be moved over to the [testing directory](https://github.com/Col-E/Java2HTML/blob/master/src/test/java/) in the form of JUnit tests.

Reply:
I have moved this into the test directory. But note it's not intended as a unit test. It would be bad practice if it was because it reads from disk.

Reply:
Good point. The content of the file can be mocked so that its not truely IO dependent though.